packages/@tailwindcss-upgrade/src/template/codemods/arbitrary-value-to-bare-value.ts

```diff
@@ -12,6 +13,74 @@ export function arbitraryValueToBareValue(
   for (let candidate of parseCandidate(rawCandidate, designSystem)) {
     let clone = structuredClone(candidate)
     let changed = false
+
+    // Convert font-stretch-* utilities
+    if (
+      clone.kind === 'functional' &&
+      clone.value?.kind === 'arbitrary' &&
+      clone.value.dataType === null &&
+      clone.root === 'font-stretch'
+    ) {
+      if (clone.value.value.endsWith('%')) {
+        let percentage = parseFloat(clone.value.value)
+        if (percentage >= 50 && percentage <= 200) {
+          changed = true
+          clone.value = {
+            kind: 'named',
+            value: clone.value.value,
+            fraction: null,
+          }
```

Comment:
`font-stretch-50.5%` _does_ work, but don't know if that should though.... wrote something on Discord about it because it feels a bit inconsistent compared to let's say `opacity-50%` (doesn't do anything), but `opacity-50` _does_ 😅

<img width="862" alt="image" src="https://github.com/user-attachments/assets/86cf0610-278b-47d4-abdb-d3d924a0555c">

Reply:
Alright, let's clean that up. Created a separate PR to solve the actual issue in the utility: https://github.com/tailwindlabs/tailwindcss/pull/14670